runtime/build.gradle

```diff
@@ -20,3 +20,12 @@ kotlin {
         }
     }
 }
+
+tasks.withType(Jar).named(kotlin.jvm().artifactsTaskName) {
```

Comment:
Please drop a **detailed** comment on what is this, how it is used and why it's important.
Otherwise, its purpose can be lost in a year or so

Reply:
Also, it would be nice to have an integration test that verifies the version is present in the manifest